### PR TITLE
fix(driver-adapters): test decimal/precision on pg/Neon

### PR DESCRIPTION
**Pull request by jkomyno**
This PR closes https://github.com/prisma/team-orm/issues/384 for Neon and Pg.

### CODE REVIEW

File: packages/cli/src/Init.ts [hunk 1]
```diff
@@ -144,13 +144,13 @@ export class Init implements Command {
...
-  Set up a new Prisma project and specify \`prisma-client-go\` as the generator provider to use
+  Set up a new Prisma project and specify prisma-client-go as the generator provider to use
     ${dim('$')} prisma init --generator-provider prisma-client-go
 
-  Set up a new Prisma project and specify \`x\` and \`y\` as the preview features to use
+  Set up a new Prisma project and specify x and y as the preview features to use
     ${dim('$')} prisma init --preview-feature x --preview-feature y
 
-  Set up a new Prisma project and specify \`./generated-client\` as the output path to use
+  Set up a new Prisma project and specify ./generated-client as the output path to use
```
Comment (Jolg42) [inline packages/cli/src/Init.ts:153]: Why did you need to revert this? It's unrelated here

**Review by Jolg42 — CHANGES_REQUESTED** [review]
I want to know why a revert was needed on something unrelated before merging this.

File: packages/cli/src/Init.ts (hunk 1 above)
Comment (jkomyno) [reply packages/cli/src/Init.ts:153]: I didn't touch anything on `cli/src/Init.ts` on purpose. I've literally run `git REVERT $MY_PREVIOUS_COMMIT`. Looks like something caught up in git's revert process.

**Review by miguelff — APPROVED** [review]
This is actually correct,  it provides more test fidelity, so I'm inclined to merge it.

File: packages/cli/src/Init.ts (hunk 1 above)
Comment (Jolg42) [reply packages/cli/src/Init.ts]: Ok, weird, thanks for fixing this.

**Review by Jolg42 — APPROVED** [review]
